snmp/tests/test_unit.py

```diff
@@ -223,6 +223,21 @@ def test_duplicate_sysobjectid_error():
     SnmpCheck('snmp', init_config, [instance])
 
 
+def test_mapping_sysobjectids():
```

Comment:
Not sure I understand "mapping" in the name of this test, did you mean to refer to an "array/list of sysobjectids"? So eg:

```suggestion
def test_sysobjectid_list():
```